dotnet/src/Connectors/Connectors.Amazon/Connectors.Amazon.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net8.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+    </PropertyGroup>
+    <PropertyGroup>
+      <NoWarn>$(NoWarn);SKEXP0001,SKEXP0010</NoWarn>
```

Comment:
Instead of commenting out tags, I found [this link](https://github.com/microsoft/semantic-kernel/blob/main/dotnet/docs/EXPERIMENTS.md) which said to add that diagnostic ID to ignore warnings. The SKEXP0001 NoWarn is to suppress Semantic Kernel core features, which without it would error in [`[FunctionCallContent.cs]`](https://github.com/microsoft/semantic-kernel/blob/main/dotnet/src/SemanticKernel.Abstractions/Contents/FunctionCallContent.cs) if not used; `FunctionCallContent.cs` is called from [ModelDiagnostics.cs](https://github.com/microsoft/semantic-kernel/blob/main/dotnet/src/InternalUtilities/src/Diagnostics/ModelDiagnostics.cs) in `StartCompletionActivity` which is used for all chat completions - Google and others use same NoWarn tag. (This NoWarn tag is also used to suppress warnings for embedding and image services though that is only relevant in feature-connectors-amazon-text-embedding and feature-connectors-amazon-text-to-image branches). Good point about SKEXP0010 though, will remove that since that is just for OpenAI and Azure OpenAI services which we don't take dependency on.